Makefile

```diff
@@ -83,6 +70,10 @@ define build
 	$(CARGO) $(CARGO_OPTS) $(1) $(CARGO_CMD_OPTS)
 endef
 
+define build_ebpf
+
+endef
```

Comment:
What does this do?

Reply:
really?!? :)
Sorry, it probably was too late yesterday and I forgot to refresh this hunk removal.